nautobot/docs/development/best-practices.md

```diff
@@ -8,12 +8,12 @@ While there are many different development interfaces in Nautobot that each expo
 - All BulkEditForm classes should inherit from `nautobot.extras.forms.NautobotBulkEditForm` (for models that support change-logging, custom fields and relationships).
 - All FilterForm classes should inherit from `nautobot.extras.forms.NautobotFilterForm` (for models that support change-logging, custom fields and relationships).
 - All FilterSet classes should inherit from either `nautobot.utilities.filters.BaseFilterSet` (for bare-bones models) or `nautobot.extras.filters.NautobotFilterSet` (for models that support change-logging, custom fields and relationships).
-- All Serializer classes should inherit from `ValidatedModelSerializer`.
+- All Serializer classes should inherit from either `NautobotModelSerializer` (for models that support custom fields and relationships, _added in version 1.4.0_), `ValidatedModelSerializer` (for writable nested serializers and more bare-bones models), or `BaseModelSerializer` (for read-only serializers and extremely minimal models).
```

Comment:
Will `ValidatedModelSerializer` work as a drop-in replacement for `WritableNestedSerializer`? If so, should `WritableNestedSerializer` be deprecated?

Reply:
No, `WritableNestedSerializer` adds functionality and should always be used for nested serializers. Clearly my thinking was a bit muddled here - I'll try to come up with some more accurate wording that reflects our best practices.